### ISSUE TITLE
Level: Red Army

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#493]
```
/ *\       
# ##       
#**t    Q  
#### ###   
O         #
###########
:*=yt
:*=yt
:*=yt
:bash=1
:block=1
:bridge=1
:brolly=1
:climb=1
:dig=1
:explode=1
:name=Red Army
:num_rabbits=1
```

**Comment by colonelfazackerley** [comment]
C5u$T|uVF$"Fd;]C5u]9w99x959;&9awF^h@TahE%;9CF5h9&99hufu

took about 10 attempts. a pleasing intro to rabbots.

**Comment by andybalaam** [comment]
Woah I haven't even tried this out myself yet! I probably should have mentioned that...

**Comment by andybalaam** [comment]
I made it using the prototype http://artificialworlds.net/rabbit-escape/level-editor/

**Comment by tttppp** [comment]
That is awesome - getting the editor working in the app would be really cool too - to allow easy playback.

**Comment by andybalaam** [comment]
My solution is similar to @colonelfazackerley 's : `MT[{unRnwv%*{!DdnROFnnRHFnT^0FLQ^^F[H=DnOujH-#M"HRQJ^={[x$On[u`

**Comment by tttppp** [comment]
My solution is a little simpler:
```
|pFOZ}O:imApU:ZA#Upb)bmU[UUdi9`
```

**Comment by andybalaam** [comment]
Nice solution - I don't think we need to change the level in response.

**Comment by andybalaam** [comment]
Released in 0.11
